src/main/java/org/apache/commons/io/input/CloseShieldInputStream.java

```diff
@@ -50,4 +54,16 @@ public void close() {
         in = ClosedInputStream.CLOSED_INPUT_STREAM;
     }
 
+    /**
+     * Creates a proxy that shields the given input stream from being
+     * closed.
+     *
+     * @param in underlying input stream
```

Comment:
`@param in underlying input stream` -> `@param inputStream the input stream to wrap`

Reply:
Changed as suggested